mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxCounters.java

```diff
@@ -87,6 +87,28 @@ public long getUnseen() {
         return unseen;
     }
 
+    public MailboxCounters sanitize() {
```

Comment:
why it's a method? Do we want to not call it sometimes?

Reply:
Yes, we don't want it when we call RecomputeMailboxCountersService as it can hide underlying inconsistencies

Reply:
I don't really like that temporal coupling. Can't we fix that?

Reply:
You mean: having 2 types?

Reply:
Yes